pkgs/games/eternity-engine/default.nix

```diff
@@ -1,20 +1,21 @@
-{ lib, stdenv, cmake, libGL, SDL, SDL_mixer, SDL_net, fetchFromGitHub, makeWrapper }:
+{ lib, stdenv, cmake, libGL, SDL2, SDL2_mixer, SDL2_net, fetchFromGitHub, makeWrapper }:
 
 stdenv.mkDerivation rec {
   pname = "eternity-engine";
-  version = "3.42.02";
+  version = "4.02.00";
   src = fetchFromGitHub {
     owner = "team-eternity";
     repo = "eternity";
     rev = version;
-    sha256 = "00kpq4k23hjmzjaymw3sdda7mqk8fjq6dzf7fmdal9fm7lfmj41k";
+    sha256 = "0dlz7axbiw003bgwk2hl43w8r2bwnxhi042i1xwdiwaja0cpnf5y";
+    fetchSubmodules = true;
   };
 
   nativeBuildInputs = [ cmake makeWrapper ];
-  buildInputs = [ libGL SDL SDL_mixer SDL_net ];
+  buildInputs = [ libGL SDL2 SDL2_mixer SDL2_net ];
 
   installPhase = ''
-    install -Dm755 source/eternity $out/lib/eternity/eternity
+    install -Dm755 eternity/eternity $out/lib/eternity/eternity
     cp -r $src/base $out/lib/eternity/base
     mkdir $out/bin
     makeWrapper $out/lib/eternity/eternity $out/bin/eternity
```

Comment:
Please update the homepage to https://doomworld.com/eternity/

Reply:
Done.